### PR TITLE
Arc-report updates

### PR DESCRIPTION
**Pull request by segeljakt**

### CODE REVIEW

**Comment by segeljakt** [comment]
After this I will bring the Overleaf doc up-to-date. Seif wants to contribute 👀

File: .github/continuous-integration.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ if [[ -d "${SCCACHE_DIR}" ]]; then
     echo "It contains $(du -hs ${SCCACHE_DIR} | cut -f1)"
 else
     echo "Creating Sccache directory at ${SCCACHE_DIR}"
-    mkdir -p ${SCCACHE_DIR}
+    sudo mkdir -p ${SCCACHE_DIR}
```
Comment (frej) [inline .github/continuous-integration.sh:43]: This hung before and is fishy. If the build doesn't run as root and you create the sccache dir as root, the build will not be allowed to write to it (unless you change permissions or ownership). Also there should be no need for sudo as `${SCCACHE_DIR}` is in the arc-runners home directory (are files/directories in the home directory owned by root due to a previous run where the user was still root?).

**Review by frej — CHANGES_REQUESTED** [review]
Suspicious about the use of sudo in the script.

I haven't reviewed the text parts, that's easier to do on the built document, but that shouldn't delay merging it.

File: .github/continuous-integration.sh (hunk 1 above)
Comment (segeljakt) [reply .github/continuous-integration.sh]: Hmm, before it did not compile without sudo. I can try remove it and see what happens

Comment (segeljakt) [reply .github/continuous-integration.sh]: wait, huh, I can't find these changes

Comment (frej) [reply .github/continuous-integration.sh]: Me neither, looks like I reviewed something stale.

**Review by frej — APPROVED** [review]

File: .github/continuous-integration.sh (hunk 1 above)
Comment (frej) [reply .github/continuous-integration.sh]: It's gone now, so LGTM.

**Pull request merged**